spec/index.bs

```diff
@@ -8327,7 +8343,7 @@ will affect a render pass's {{GPURenderPassDescriptor/depthStencilAttachment}}:
             |descriptor|.{{GPUDepthStencilState/stencilBack}} are not the default values:
             - |descriptor|.{{GPUDepthStencilState/format}} must have a stencil component.
 
-    Issue: how can this algorithm support depth/stencil formats that are added in extensions?
+    Issue: <label data-milestone="post-V1"> how can this algorithm support depth/stencil formats that are added in extensions?
```

Comment:
`<label>` is normally used for associating text with form elements. Maybe `<span>`...?
Actually, since this attribute holds actual content we're going to inject later anyway, why not `<a class=milestone>post-V1</span>` or something? We can still generate the links based on the `innerText` just as well.

Actually, what if we just defined `<dfn dfn>Target: post-V1</dfn>` in the anchors list so that it links to the milestone?